source/utils/date.cpp

```diff
@@ -26,10 +26,14 @@
 #include <xlnt/utils/date.hpp>
 #include <detail/time_helpers.hpp>
 
+/// Invalid weekday for checking whether std::mktime was successful - see below.
+/// Must be outside of the range [0, 6].
+constexpr int INVALID_WDAY = -1;
```

Comment:
Should we add this to an unnamed namespace, to ensure we never violate the one definition rule?

Reply:
Could this really ever be an ODR violation? To my knowledge, this variable should only have internal linkage, thus it should only be available in the translation unit of `date.cpp` (note: not a header file!). Personally, I never had issues with such variables in my project, but feel free to prove me wrong :sweat_smile:

Reply:
I often use unnamed namespaces anyway, so I don't need to keep in mind all the default linkage rules. But indeed, not needed for a const expr.